ui/src/app/applications/components/application-pod-view/pod-view.tsx

```diff
@@ -138,6 +138,28 @@ export class PodView extends React.Component<PodViewProps> {
                                                         </div>
                                                     ) : (
                                                         <div className='pod-view__node__info'>
+                                                            {
+                                                                <Tooltip
+                                                                    content={
+                                                                        <React.Fragment>
+                                                                            <div className='row'> Requests: </div>
+                                                                            <div className='row'>
+                                                                                <div className='columns small-6' style={{whiteSpace: 'nowrap'}}>
+                                                                                    CPU:
+                                                                                </div>
+                                                                                <div className='columns small-6'>{GroupPodResources.cpuRequest}</div>
+                                                                            </div>
+                                                                            <div className='row'>
+                                                                                <div className='columns small-6' style={{whiteSpace: 'nowrap'}}>
+                                                                                    Memory:
+                                                                                </div>
+                                                                                <div className='columns small-6'>{GroupPodResources.memoryRequests}</div>
+                                                                            </div>
+                                                                        </React.Fragment>
+                                                                    }>
+                                                                    <div id='cost'> Requests? </div>
```

Comment:
Remove the `?`

Reply:
ok sure, I kept it so it looks more interactive.